website_compassion/models/product_template.py

```diff
@@ -0,0 +1,20 @@
+##############################################################################
+#
+#    Copyright (C) 2021 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Jonathan Guerne <jonathan.guerne@gmail.com>
+#
+#    The licence is in the file __manifest__.py
+#
+##############################################################################
+
+import logging
+
+from odoo import models, api, fields
+
+
+class ProductProduct(models.Model):
+    _name = "product.template"
+    _inherit = "product.template"
+
+    website_published = fields.Boolean(index=True)
```

Comment:
The correct way is to inherit from an abstract class already implemented for this purpose:
```suggestion
    _name = "product.template"
    _inherit = ["product.template", "website.published.mixin"]
```